File: Java/api/org/rr0/im/business/event/circumstance/Precipitations.java
package org.rr0.im.business.event.circumstance;

import org.rr0.im.service.function.classification.Classifiable;
/**
 * 
 *
 * @author Jérôme Beau
 * @version 10 mai 2003 17:04:26
 */
public interface Precipitations extends Classifiable {
    int getMilimeters ();

    /**
     * @return The expected visibility given that Precipitations
     */
    float getVisibility();

    /**
     * @return The expected humidity given that Precipitations
     */
    float getHumidity();
}

File: api/src/org/rr0/im/business/event/circumstance/Precipitations.ts
import {Classifiable} from "../../../service/function/classification/Classifiable";

export interface Precipitations extends Classifiable {
  getMilimeters (): number;

  /**
   * @return The expected visibility given that Precipitations
   */
  getVisibility(): number;

  /**
   * @return The expected humidity given that Precipitations
   */
  getHumidity(): number;
}
